main/getting-started/ertp-introduction.md

```diff
@@ -12,83 +12,93 @@ object. If it doesn't have a reference, it can't.
 imaginary currencies.
 
-If everything passes the checks, the asset moves from the payment to
-the purse. If there's a problem, it throws an error.
+If everything passes the checks, the asset moves from the `payment` to
+the `purse`. If there's a problem, it throws an error.
 
 After a successful deposit, ERTP guarantees:
-- The payment object is empty.
-- The purse contains the payment's full content.
+- The `payment` is burned (i.e. destroyed)..
+- The `purse` contains the total of what it held before plus the `payment`'s full content.
+  - i.e. If the `purse` had 7 Quatloos and the `payment` had 3 Quatloos, after depositing the `payment`
+    the `purse` has 10 Quatloos.
 
-When the `deposit` call throws an error (i.e. something went wrong),
+When the `deposit()` call throws an error (i.e. something went wrong),
 ERTP guarantees: 
-- The alleged payment is in the same state as before the call.
-- The purse is in the same state as before the call.
+- The alleged `payment` is in the same state as before the call.
+- The `purse` is in the same state as before the call.
+
+In other words, a failed attempt to deposit a 3 Quatloo `payment`
+in a 7 Quatloo `purse` means the `payment` continues to exist and hold
+3 Quatloos, and the `purse` continues to hold 7 Quatloos.
 
 ## Issuers and mints
 
-Other key ERTP objects are:
+Other key ERTP components are:
 
-- **Mints**: Issue new digital assets as a new Payment. Mints only
-issue one type of asset (quatloos, simoleons, moola, etc. Note that these
+- **Mints**: Issue new digital assets as a new `Payment`. `Mints` only
+issue one kind of asset (Quatloos, Simoleons, Moola, etc. Note that these
 are all imaginary currencies used as examples.) We refer to
-that type as the mint's *Brand*. So if a mint issues quatloos, it's a
-quatloo brand mint.  Only mints can issue new digital assets. To mint
-new assets of a particular type, you must have a reference to that
-type's mint 
-
-- **Issuers**: Create empty purses and payments and map minted
- assets to them when assets are added or removed. 
- Issuers verify and move digital assets.
-
-An issuer's special admin facet is a Mint, and that Mint and Issuer are
- associated with each other. With a reference to an issuer, you can
- check the validity of a payment in that issuer's mint's assets, as
- well as claim it either as a new payment to yourself or a purse you
- control. 
-
-Issuers (i.e. their references) should be gotten from a trusted source
-and then relied upon as the decider of whether an untrusted payment is
+that kind as a `mint`'s *Brand*. So if a `mint` issues Quatloos, it's a
+Quatloo `brand` `mint`.  Only `mints` can issue new digital assets. To mint
+new assets of a particular kind, you must have a reference to that
+kind's `mint` 
+
+- **Issuers**: Create empty `purses`, manipulate and operate on
+ `payments` and map minted assets to both of them when assets are 
+ added or removed. `Issuers` verify and move digital assets.
```

Comment:
I'm not sure what "map minted assets to both of them " means here. Does this refer to split() and combine()? it could be clearer.